myadmin-service-impls/student-services/src/main/java/za/ac/unisa/myadmin/student/services/impls/ModuleEnrolmentServiceImpl.java

```diff
@@ -5,11 +5,7 @@
 import java.time.LocalDate;
 import java.time.ZoneId;
 import java.time.format.DateTimeFormatter;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.Date;
-import java.util.List;
-import java.util.Optional;
+import java.util.*;
```

Comment:
wildcard imports can cause ambiguity. Standard java packages are ok I guess.
I'm not bothered by it, Think I setup my IDE not to.